### PR TITLE
cloud: fix wrong command in dumpling export doc

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

close https://github.com/pingcap/docs/issues/11247

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 1]
```diff
@@ -36,18 +36,38 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup install dumpling
+    tiup install dumpling:v6.1.1
     ```
 
-4. Export your data using Dumpling from TiDB.
+4. Export your data using Dumpling from TiDB. You can export data in the following formats:
```
Comment (ran-huang) [inline tidb-cloud/export-data-from-tidb-cloud.md:42]:
```suggestion
4. Export your data using Dumpling from TiDB. 
```

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 2]
```diff
@@ -36,18 +36,38 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup install dumpling
+    tiup install dumpling:v6.1.1
     ```
 
-4. Export your data using Dumpling from TiDB.
+4. Export your data using Dumpling from TiDB.
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:42]:
```suggestion
4. Export your data using Dumpling from TiDB. 

    You can get the following connection parameters `${tidb-endpoint}`, `${port}`, and `${user}` from the **Connection** section on the **Clusters** page.
```

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 3]
```diff
@@ -36,18 +36,38 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
...
+    tiup dumpling:v6.1.1 -h ${tidb-endpoint} -P ${port} -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
+    ```
+
+    </div>
+
+    Parameters are described as follows:
+
+    - `-h`: The TiDB cluster endpoint.
+    - `-P`: The TiDB cluster port.
+    - `-u`: The TiDB cluster user.
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:65]:
```suggestion
    - `-u`: The TiDB cluster user.
    - `-p`: The TiDB cluster password.
    - `-F`: The maximum size of a single file.
```

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 4]
```diff
@@ -36,18 +36,38 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
...
+    </div>
+    <div label="Dedicated Tier">
+
+    ```shell
+    tiup dumpling:v6.1.1 -h ${tidb-endpoint} -P ${port} -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
+    ```
+
+    </div>
+
+    Parameters are described as follows:
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:61]:
```suggestion
    Options are described as follows:
```

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 5]
```diff
@@ -36,18 +36,43 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
...
     ```shell
-    tiup install dumpling
+    tiup install dumpling:v6.1.1
     ```
 
-4. Export your data using Dumpling from TiDB.
+4. Export your data using Dumpling from TiDB. 
 
-    {{< copyable "shell-regular" >}}
+    You can get the following connection parameters `${tidb-endpoint}`, `${port}`, and `${user}` from the **Connection** section on the **Clusters** page.
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:44]:
```suggestion
    You can get the following connection parameters `${tidb_endpoint}`, `${port}`, and `${user}` from the **Connection** section on the **Clusters** page.
```

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 6]
```diff
@@ -36,18 +36,43 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
...
+
+    <SimpleTab>
+
+    <div label="Serverless Tier">
+
+    ```shell
+    tiup dumpling:v6.1.1 -h ${tidb-endpoint} -P 4000 -u ${user} -p ${password} --ca=${cert_path} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:51]:
```suggestion
    tiup dumpling:v6.1.1 -h ${tidb_endpoint} -P 4000 -u ${user} -p ${password} --ca=${ca_path} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
```

File: tidb-cloud/export-data-from-tidb-cloud.md [hunk 7]
```diff
@@ -36,18 +36,43 @@ You can use the tool [Dumpling](https://github.com/pingcap/dumpling) for data ex
...
+    ```
+
+    </div>
+    <div label="Dedicated Tier">
 
     ```shell
-    tiup dumpling -h ${tidb-endpoint} -P 3306 -u ${user} -F 67108864 -t 4 -o /path/to/export/dir
+    tiup dumpling:v6.1.1 -h ${tidb-endpoint} -P ${port} -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
```
Comment (Oreoxmt) [inline tidb-cloud/export-data-from-tidb-cloud.md:58]:
```suggestion
    tiup dumpling:v6.1.1 -h ${tidb_endpoint} -P ${port} -u ${user} -p ${password} -F 67108864MiB -t 4 -o ${export_dir} --filetype sql
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**